src/jws/flattened.rs

```diff
@@ -0,0 +1,542 @@
+//! Flattened JWS signatures: see RFC 7515 section 7.2.2
```

Comment:
Please provide some examples on how flattened JWS can be used with the library.

Reply:
Done